### PR TITLE
[ADD] currency_rate_update: daily spot rates from Bank of England

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: currency_rate_update/model/#company.py# [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
```
Comment (yvaucher) [inline currency_rate_update/model/#company.py#]: This file seems to be a backup from your IDE please remove it

**Comment by yvaucher** [comment]
On what have you based your PR? Is it really for 9.0 version ? 

In that case please rebase it on #318

**Comment by ghost** [comment]
Following yvaucher's suggestion, I would like to rebase on [PR #318](https://github.com/OCA/account-financial-tools/pull/318). 
Please advise as to how this should be done.

**Comment by adrienpeiffer** [comment]
I'll close this one as the user has been removed.

**Pull request closed without merging**